utils/bump-version.sh

```diff
@@ -16,5 +16,6 @@ function inplace_sed {
   rm "$2$suffix"
 }
 
+echo "${VER}" > VERSION
```

Comment:
the `VERSION` file is what is used in `setup.py` -- tiny somewhat unrelated bugfix here.